.travis.yml

```diff
@@ -1,19 +0,0 @@
-language: node_js
-node_js:
-- 10
```

Comment:
The old CI was using Node 10, while the new one is at 18. This is likely the cause of failures. You'll likely need to update dependencies for Node 18 to work.